sycl/include/CL/sycl/detail/defines_elementary.hpp

```diff
@@ -87,10 +87,14 @@
 #endif
 #endif // __SYCL_FALLTHROUGH
 
+// Stringify an argument to pass it in _Pragma directive below.
+#ifndef __STRINGIFY
+#define __STRINGIFY(x) #x
+#endif // __STRINGIFY
```

Comment:
This duplicates `__SYCL_STRINGIFY_LINE_HELP`.
Please, update `common.hpp` and `config.cpp` and remove duplicate.

BTW, why `__STRINGIFY` and not `__SYCL_STRINGIFY`?

Reply:
Applied, thanks!